src/getAdminLayers.js

```diff
@@ -10,14 +10,16 @@
  */
 function getAdminLayers(layer) {
   switch (layer) {
+    case 'macroregion':
+        return ['macroregion'];
     case 'country':
-        return ['country'];
+        return ['macroregion', 'country'];
     case 'region':
-        return ['country', 'macroregion', 'region'];
+        return ['macroregion', 'country', 'region'];
     case 'county':
-        return ['country', 'macroregion', 'region', 'macrocounty', 'county'];
+        return ['macroregion', 'country', 'region', 'macrocounty', 'county'];
     case 'locality':
-        return ['country', 'macroregion', 'region', 'macrocounty', 'county', 'locality', 'borough'];
+        return ['macroregion', 'country', 'region', 'macrocounty', 'county', 'locality'];
```

Comment:
if these are all intended to be in order of nesting (as per hierarchy) wouldn't macroregion be after country? https://github.com/whosonfirst/whosonfirst-placetypes#here-is-a-pretty-picture